### ISSUE TITLE
Please release the updates you have

### ISSUE THREAD

**Issue by rossdargan** [mattttvaughn/chronicle#27]
You have made some great changes/updates, but the last update in the store is from June 2020. It's a great app and adding things like android auto would be great!

I'm a c# dev but happy to help out where I can. Are there any of the issues you would like me to look at?

**Comment by hskrtich** [comment]
@mattttvaughn What do we need to do to get this update release?

**Comment by mattttvaughn** [comment]
@bskrtich at the moment, no in-app changes needed that I can think of.

I need to get my test server online for Play Store reviewers since Auto is enabled and they usually manually test that. If they find any issues then I'll write some fixes or post an issue here.

And eventually we'll/I'll need to test data migrations from the play store versions of the app to the current version. There's been some changes to db schema, will need to ensure migrations are working correctly

**Comment by oucil** [comment]
+1 for getting the update to the play store. This isn't meant to be cynical, just a reminder that you're doing all this great work and those of us that paid for the app are still on 0.45.1.  There are no instructions on the project page here regarding installing the APK manually, and having tried it a few times, it won't overwrite the Play Store version on the phone, should we uninstall and start fresh here?  Are there issues with that?  Would just really appreciate some love thrown the way of those of us that supported your early work.

**Comment by mattttvaughn** [comment]
It looks like issue #38 affects the current play store release, so will have to push this fix ASAP to avoid new users running into  blocking behavior on login.

Here's current progress towards a release:

- [X] ~~Update the IAP billing library~~
- [X] ~~Fix #38~~
- [X] ~~Test server up and working~~
- [x] Export and upload the 0.51.2 apk to github
- [x] Upload release to internal testing track
- [x] Play Store approval for Auto (approved for internal testing, not sure if they will review again for beta track)
- [x] Update play store privacy policy. (_should_ be simple since app doesn't have any analytics or off-device storage. but could be a whole thing if it needs to account for Plex's server-side data handling)
- [x] Manually test data migrations from 0.45.1 to 0.52.2
- [x] Promote update to beta track
- [ ] Promo update to release track

Going to avoid fixing #38 as a hotfix and just focus on this release since it's ready to go.

**Comment by hskrtich** [comment]
@mattttvaughn I think it would be helpful to try to automate some of the release process to help speed this up in the future and then it would require less of your time.

**Comment by mattttvaughn** [comment]
@bskrtich agreed. setting up CI seems like it'd be a fairly good value gain. I'll be keeping my eyes on that in the near future
